src/control-plane/backend/lambda/api/service/metadata.ts

```diff
@@ -114,6 +119,44 @@ export class MetadataEventServ {
       next(error);
     }
   };
+
+  public async trigger(req: any, res: any, next: any) {
+    try {
+      const { projectId } = req.body;
+      const trigger = await clickStreamStore.isManualTrigger(projectId);
```

Comment:
@dengmingtong Let's only trigger the scanning for the specific application, not all applications in the project.

Reply:
pushed the commit to support this feature.